src/test/java/seedu/address/logic/commands/AddVolunteerCommandTest.java

```diff
@@ -141,6 +143,16 @@ public void setPerson(Person target, Person editedPerson) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public Elderly getElderly(Nric nric) {
```

Comment:
Just a heads up: @Zxun2 is planning to abstract the stub out to its own class in #32. It's a minor conflict, but good to note.

Reply:
Thanks! Will update my code once @Zxun2 's code is merged